src/osemgrep/cli_scan/Rule_filtering.ml

```diff
@@ -15,12 +15,30 @@
 (*****************************************************************************)
 (* Types *)
 (*****************************************************************************)
-type conf = { exclude_rule_ids : Rule_ID.t list; severity : Rule.severity list }
+type product = SAST | SCA | Secrets | Interfile [@@deriving show]
```

Comment:
why not using the types in semgrep_output_v1.atd? :( product is already defined there ...